fabmukunzi/certificate-manager---Fabrice
# Task2 KAN-47: Certificates table

#### PR description

- Adds certificates table

#### Ticket / Jira Link

- [KAN-47](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?selectedIssue=KAN-47)

#### Checklist:


#### How To Test Changes

- Checkout to this branch `feature/task2-KAN-47-certificates-table`
- Type `npm i` to install packages
- Open the terminal and type `npm run dev` to run the app
- Open your browser and navigate to `http://localhost:3000/`
- On the sidebar, click on example 1 and you should see a table with certificates data

#### Screenshots:
![image](https://github.com/user-attachments/assets/f93b774e-ee17-48b0-8cc5-8570d2b055a7)  ![image](https://github.com/user-attachments/assets/74c9670f-0431-429d-b18b-be1371aa73c8)
